src/points_set.py

```diff
@@ -97,25 +99,37 @@ def remove_point(self, point):
             self._points_to_add.add(point)
 
         self._points_to_remove = {p for p in self._points if self._can_remove(p)}
+        self._value = self._get_value(len(self._points), self._triangles)
 
-    def _get_value(self, points_n, area):
-        if area == 0:
+    def _get_value(self, points_n, triangles):
```

Comment:
Skoro value i area to dwie różne rzeczy, to ta funkcja chyba nie może się tak nazywać zwracając area. Już nie wiem co jest czym.